ElasticsearchReportHelper.php

```diff
@@ -846,13 +950,19 @@ public static function userFilters(array $options) {
       'definesPermissions' => FALSE,
       'sharingCode' => 'R',
     ], $options);
-    $filterData = report_filters_load_existing($options['readAuth'], $options['sharingCode'], TRUE);
+
+    // Sharing code can be specified as a comma separated list of codes
+    $sharingCodes = explode(',', $options['sharingCode']);
```

Comment:
Will need to be mentioned in documentation.

Reply:
I've made a note to do it.